tests/Integration/Routing/ImplicitBindingTest.php

```diff
@@ -23,32 +22,44 @@ protected function getEnvironmentSetUp($app)
         ]);
     }
 
-    protected function setUp(): void
+    protected function defineDatabaseMigrations(): void
     {
-        parent::setUp();
-
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name');
             $table->timestamps();
         });
+
+        $this->beforeApplicationDestroyed(function () {
+            Schema::dropIfExists('users');
+        });
     }
 
     public function testPreviousUrlWithoutSession()
     {
-        $user = ImplicitBindingModel::create(['name' => 'Dries']);
+        $route = <<<PHP
+<?php
 
-        Route::post('/user/{user}', function (ImplicitBindingModel $user) {
-            dd($user);
+use Illuminate\Tests\Integration\Routing\ImplicitBindingModel;
 
-            return $user;
-        })->middleware('web');
+Route::post('/user/{user}', function (ImplicitBindingModel \$user) {
+    return \$user;
+})->middleware('web');
+PHP;
+        file_put_contents(base_path('routes/testbench.php'), $route);
```

Comment:
In order to cache route with testbench, we need to allow it to get loaded from `laravel/bootstrap/app.php`. The easiest option I can come out right now is to write a route file for cache to work.